src/Core/Module/ModuleManager.php

```diff
@@ -90,6 +90,22 @@ public function __construct(
         $this->hookManager = $hookManager;
     }
 
+    public function upload(string $name, string $source): void
```

Comment:
I believe we discussed this but it seems the `$name` is not useful here, so if it's not used at all it should be removed (and also in the CQRS command)

Unless we decide to use it for module validation as we discussed quickly?